src/pages/charter.js

```diff
@@ -0,0 +1,309 @@
+      </ol> 
+      <h2>6) Subsidiaries and Related Companies</h2>
+      <ol>
+        <li>Definitions:
+        <ol>
+          <li>“Subsidiaries” means any entity in which a Member owns, directly or indirectly, more
+          than fifty percent of the voting securities or membership interests of the entity in
+          question;</li>
+          <li>“Related Company” means any entity which controls or is controlled by a Member or
+          which, together with a Member, is under the common control of a third party, in each
+          case where such control results from ownership, either directly or indirectly, of more
+          than fifty percent of the voting securities or membership interests of the entity in
+          question; and</li>
+          <li>“Related Companies” are entities that are each a Related Company of a Member.</li>
+        </ol>
+        </li>
+        <li>Only the legal entity which has executed a Participation Agreement and its Subsidiaries
+        will be entitled to enjoy the rights and privileges of such Membership; provided,
+        however, that such Member and its Subsidiaries will be treated together as a single
+        Member.</li>
+        <li>If a Member is itself a foundation, association, consortium, open source project,
+        membership organization, user group or other entity that has members or sponsors, then
+        the rights and privileges granted to such Member will extend only to the
+        employee-representatives of such Member, and not to its members or sponsors, unless
+        otherwise approved by the Governing Board in a specific case.</li>
+        <li>Directed Fund Membership is non-transferable, non-salable and non-assignable, except a
+        Member may transfer its current Membership benefits and obligations to a successor of
+        substantially all of its business or assets, whether by merger, sale or otherwise; provided
+        that the transferee agrees to be bound by this Charter and the Bylaws and policies
+        required by LF membership.</li>
+      </ol>
+      <h2>7) Good Standing</h2>
+      <ol>
+        <li>The Linux Foundation’s Good Standing Policy is available at <a href="https://www.linuxfoundation.org/good-standing-policy" target="_blank" rel="noopener noreferrer">https://www.linuxfoundation.org/good-standing-policy</a> and will apply to Members of this Directed Fund.</li>
+      </ol>
+      <h2>8) Trademarks</h2>
+      <ol>
+        <li>Any trademarks relating to the Directed Fund or the Technical Projects<a href="#clarification-1"><sup>1</sup></a> , including
+        without limitation any mark relating to any conformance program, must be transferred to
+        and held by LF Projects, LLC or the Linux Foundation and available for use pursuant to
+        LF Projects, LLC’s trademark usage policy, available at <a href="www.lfprojects.org/trademarks/" target="_blank" rel="noopener noreferrer">www.lfprojects.org/trademarks/</a>.</li>
```

Comment:
This link seems broken in the preview, it misses the `https://` prefix.

Reply:
Thanks for pointing that out. I will update the links.